package.json

```diff
@@ -8,12 +8,6 @@
     "compile": "npx hardhat compile",
     "verify": "npx hardhat etherscan-verify",
     "test": "npx hardhat test",
-    "deploy-counter": "npx hardhat run ./scripts/1-deploy-counter.js --network ropsten",
-    "deploy-resolver": "npx hardhat run ./scripts/2-deploy-counter-resolver.js --network ropsten",
-    "deposit": "npx hardhat run ./scripts/3-deposit-funds.js --network ropsten",
-    "submit-task": "npx hardhat run ./scripts/4-submit-task.js --network ropsten",
-    "cancel-task": "npx hardhat run ./scripts/5-cancel-task.js --network ropsten",
-    "withdraw": "npx hardhat run ./scripts/6-withdraw-funds.js --network ropsten",
```

Comment:
I see you are removing all those scripts, I guess the idea is that we don't need js example scripts to create / manage tasks anymore because we have 2 other alternatives:
- creattion in js => ops-sdk examples
- creation via smart contract => OpsTaskCreator examples
is that your reasoning?

Reply:
Yup that was what I was thinking